client/src/pages/Requests/requests.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import axios from 'axios';
+
+class Request extends React.Component {
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  constructor(props: {} | Readonly<{}>) {
+    super(props);
+    this.state = {
+      requestInfo: null,
+      error: '',
+    };
```

Comment:
This component really isn't needed to be class-based, we could probably do a functional component and utilise hooks for more project consistency.